app/_src/quickstart/universal-demo.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Deploy Kuma on Universal
+---
+
+To start learning how {{site.mesh_product_name}} works, you run and secure a simple demo application that consists of two services:
+
+- `demo-app`: a web application that lets you increment a numeric counter. It listens on port 5000
+- `redis`: data store for the counter
+
+
+{% mermaid %}
+---
+title: service graph of the demo app
+---
+flowchart LR
+demo-app(demo-app :5000)
+redis(redis :6379)
+demo-app --> redis
+{% endmermaid %}
+
+## Prerequisites
+
+* [Redis installed](https://redis.io/docs/getting-started/)
+* Demo app downloaded from GitHub:
+  ```sh
+  git clone https://github.com/kumahq/kuma-counter-demo.git
+  ```
+* Optional: To explore traffic metrics with the demo app, you also need to [set up Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/). See the [MeshMetric policy documentation](/docs/{{ page.version }}/policies/meshmetric/).
+
+## Install {{site.mesh_product_name}}
+
+{% tabs install useUrlFragment=false %}
+{% tab install Docker %}
+Install a {{site.mesh_product_name}} control plane using the Docker images:
+
+* **kuma-cp**: at `docker.io/{{ site.mesh_docker_org }}/kuma-cp:{{ page.version }}`
+* **kuma-dp**: at `docker.io/{{ site.mesh_docker_org }}/kuma-dp:{{ page.version }}`
+* **kumactl**: at `docker.io/{{ site.mesh_docker_org }}/kumactl:{{ page.version }}`
+
+You can freely `docker pull` these images to start using {{site.mesh_product_name}}, as we will demonstrate in the following steps.
+{% endtab %}
+{% tab install Operating Systems %}
+Do one of the following to download {{site.mesh_product_name}}:
+
+* Run the following script to automatically detect the operating system (Amazon Linux, CentOS, RedHat, Debian, Ubuntu, and macOS) and download {{site.mesh_product_name}}:
+    <div class="language-sh">
+    <pre class="no-line-numbers"><code>curl -L {{site.mesh_download_url}} | VERSION={{ page.version }} sh -</code></pre>
```

Comment:
found the original https://github.com/kumahq/kuma-website/blob/79c306e2129eb8380475549c580085f1770329dc/app/_src/production/install-kumactl.md#L27

```suggestion
  <pre class="no-line-numbers"><code>curl -L {{site.links.web}}{% if page.edition %}/{{page.edition}}{% endif %}/installer.sh | VERSION={{ page.version_data.version }} sh -</code></pre>```